lib/free/set-status.js

```diff
@@ -12,7 +12,7 @@ function setStatusFree (newStatus, context) {
     head_sha: pullRequest.head.sha,
     status: 'in_progress',
     output: {
-      title: 'Work in progress',
+      title: `Title contains ${newStatus.match === '🚧' ? 'a construction sign!' : `"${newStatus.match}"`}`,
```

Comment:
I think you forgot to remove replace "sign" with "emoji". I’d also remove the exclamation mark :) 

```suggestion
      title: `Title contains ${newStatus.match === '🚧' ? 'a construction emoji' : `"${newStatus.match}"`}`,
```